src/identity/interaction/email-password/util/RegistrationManager.ts

```diff
@@ -123,8 +123,13 @@ export class RegistrationManager {
     const trimmedEmail = this.trimString(email);
     assert(trimmedEmail && emailRegex.test(trimmedEmail), 'Please enter a valid e-mail address.');
 
+    const trimmedPassword = this.trimString(password);
```

Comment:
Do we really trim passwords? Is that a documented practice?

Reply:
It's not documented, I added it (and pod name trimming) at some point in the past to be consistent with how we already trim WebIDs.

Reply:
I think all the other trimming is fine; it's just that we don't say that spaces will be trimmed from passwords. This might wreak havoc on password managers, password length and complexity checkers, etc. So would _not_ trim password, but would trim all others.